### ISSUE TITLE
The optional information (setAdditionalInformation) text is not showing up in the Banking App

### ISSUE THREAD

**Issue by oliveratgithub** [sprain/php-swiss-qr-bill#154]
Hi there,
In the `example.php` there is an Additional Infotext, which I understand should be transferred when scanning and shown within the Banking App / on a Payment. However, I cannot get it to show up. Tried with the UBS Banking App.

```php
// Optionally, add some human-readable information about what the bill is for.
$qrBill->setAdditionalInformation(
	QrBill\DataGroup\Element\AdditionalInformation::create(
		'Invoice 123456, Gardening work'
	)
);
```

Is this an issue on the end of the Banking App and processing the QR Code information, or a bug in the Code / has the Spec for this additional Info-Text probably changed?

**Comment by oliveratgithub** [comment]
From a screenshot in Issue #61 I see that it seemed to work some time ago…

**Comment by sprain** [comment]
I just tried with my _Raiffeisen_ banking app and the newest version of this library. The additional information does appear as expected.

So there is two things to try:
* What happens if you scan a qr invoice created for example here: https://qr-rechnung.net ? If this does not work it might be an issue at UBS.
* If it works there, please provide an example qr code which does not work for you for further debugging.

Looking forward to getting to know your results :)

**Comment by oliveratgithub** [comment]
@sprain You're right – it's working. I was able to validate it on https://qr-rechnung.net/ and with my other Banking App (had to progress on to the 2nd screen to see the Additional Information).

Seems to be an issue with the UBS Banking App…

**Comment by sprain** [comment]
Great, thanks for the quick reply!

> Seems to be an issue with the UBS Banking App…

Well … 🤷🏼‍♂️
